packages/health-checks/src/health-checks.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * Represents the health state of a component or system.
  */
-export type HealthStatus = 'HEALTHY' | 'UNHEALTHY' | 'UNKNOWN';
+export type HealthStatus = 'HEALTHY' | 'UNHEALTHY' | 'TIMEDOUT';
```

Comment:
I think that the overall health status should never include `TIMEDOUT`. Individual components can time out, which should ultimately lead to an `UNHEALTHY` overall status.

Reply:
...and it looks like this is how you've implemented it in the `aggregateHealthStatus()` reducer. So I think the `TIMEDOUT` type can be removed here.

Reply:
Split these out into Overall and Component versions so it's explicit what you're dealing with since they're effectively different "enums".